test/Core.Test/OrganizationFeatures/OrganizationSubscriptionUpdate/UpdateSecretsManagerSubscriptionCommandTests.cs

```diff
@@ -376,6 +523,7 @@ public class UpdateSecretsManagerSubscriptionCommandTests
         var organization = new Organization
```

Comment:
This test validates that you can update all the values at once.

Please also add tests for updating each value individually (e.g. what happens if we're asking for an update to `SmSeats` but no other value?)

Reply:
Done